antora-ui-camel/src/css/docs.css

```diff
@@ -1,54 +1,70 @@
-.projects h2 {
+.docs h2 {
   text-align: center;
 }
 
-.projects .project {
+.camel-project {
+  width: calc(100% - 2.5rem);
```

Comment:
The width isn't calculated to be in the center of the page horizontally. Best set the width to 
`width: calc(100% - 2.2rem)`